wordpress-mobile/WordPress-iOS
# Enable Preformatted block in Gutenberg.

Enables the preformatted block in iOS for release 1.17.0

To test:
 - Start a post using Gutenberg
 - Add a preformatted block to the post
 - See if it works correctly and interacts correctly with other blocks.

Update release notes:
